interview-questions.md

```diff
@@ -1,54 +1,26 @@
 # ASSESSMENT 4: Interview Practice Questions
 Answer the following questions.
-
 First, without external resources. Challenge yourself to answer from memory.
-
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.  
-
 1. What is an instance variable in Ruby? How is it different from other variables?
-
-  Your answer:
-
-  Researched answer:
-
-
-
+  Your answer: it has a name starting with @ and its scope is specific to whatever object its referred to.
+  Researched answer: instance variables do not have to be declared and its is dynamically connected to an object when it is first referenced.
 2. What is a block in Ruby?
-
-  Your answer:
-
-  Researched answer:
-
-
-
+  Your answer: It is a piece of code that accepts arguments and returns a value
+  Researched answer: it is like a method but it doesnt have a name and does not belong to an object. they can also be created by passing them through a method and then calling that method.
 3. Ruby has an implicit return. What does that mean?
-
-  Your answer:
-
+  Your answer: it will automatically return the value of the last line in the method
   Researched answer:
-
-
-
```

Comment:
Not seeing a researched answer here. Lets be sure to check for these and provide them as the instructions detail.